File: ws/vargas/u1/ws05Dependencies/src/ec_edu_espe_chickenfarmsimulator_model/Egg.java

package ec_edu_espe_chickenfarmsimulator_model;

/**
 *
 * @author César Vargas, Paradigm,@ESPE
 */
public class Egg {
    private char size;

    @Override
    public String toString() {
        return "Egg{" + "size=" + size + '}';
    }
    
    public Egg(char size) {
        this.size = size;
    }

    /**
     * @return the size
     */
    public char getSize() {
        return size;
    }

    /**
     * @param size the size to set
     */
    public void setSize(char size) {
        this.size = size;
    }
    
}

File: hw/aymacana/u1/hw07ChickenFarmSimulator2/ChickenFarmSimulator2JS/egg.js
class Egg {
    constructor(size) {
        this._size = size;  // 'S', 'M' o 'L'
    }

    toString() {
        return `Egg{size = ${this._size}}`;
    }

    get size() {
        return this._size;
    }

    set size(size) {
        this._size = size;
    }
}

module.exports = Egg;